docs/en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -339,6 +339,46 @@ $supporters = $team->Supporters()->where(['"TeamSupporter"."Ranking"' => 1]);
 Note: ->filter() currently does not support joined fields natively due to the fact that the
 query for the join table is isolated from the outer query controlled by DataList.
 
+### Polymorphic many_many (Experimental)
+
+Using many_many through, it is possible to support polymorphic relations on the mapping table.
+Note, that this feature is currently experimental, and has certain limitations:
+ - This feature only works with many_many through
+ - This feature will only allow polymorphic many_many, but not belongs_many_many. However,
+   you can have a has_many relation to the mapping table on this side, and iterate through this
+   to collate parent records.
```

Comment:
> iterate through this to collate parent records.

Took me a couple of attempts to understand this, could you add an example further down and link this to it? Am I right in thinking it would work something like this?

```php
// Filtering by tags from the "some object" side
$taggedSomeObjects = SomeObject::get()->filter(['Tag.Name' => ['New', 'Featured']]);

// Or by collating IDs from the "tag" side
$tags = Tag::get()->filter(['Title' => ['New', 'Featured']]);
$taggedObjectIDs = $tags->relation('TagMappings')
    ->filter(['ParentClass' => SomeObject::class])
    ->column('ParentID');
$taggedSomeObjects = SomeObject::get()->byIDs($taggedObjectIDs);
```

Reply:
Yeah I can add the generator in the example.

Reply:
Added this.

```php
    public function TaggedObjects()
    {
        foreach ($this->TagMappings() as $mapping) {
            yield $mapping->Parent();
        }
    }
```